README.md

```diff
@@ -1,44 +1,51 @@
-<img align="right" src="images/logo.svg" alt="SimpleX logo" height="90">
+<h1 align="center">
+  <img src="images/simplex-chat-logo.svg" alt="SimpleX logo">
 
-# SimpleX chat
+</h1>
 
-## Private, secure, decentralized
+<p align="center">
+  <img alt="GitHub all releases" src="https://img.shields.io/github/downloads/simplex-chat/simplex-chat/total">
+  <img alt="GitHub all releases" src="https://img.shields.io/github/v/release/simplex-chat/simplex-chat">
+  <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/SimpleXChat?style=social">
+</p>
 
-[![GitHub build](https://github.com/simplex-chat/simplex-chat/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-chat/actions?query=workflow%3Abuild)
-[![GitHub release](https://img.shields.io/github/v/release/simplex-chat/simplex-chat)](https://github.com/simplex-chat/simplex-chat/releases)
+---
 
-> **NEW in v0.4: [groups](#groups) and [sending files](#sending-files)!**
+# SimpleX Chat
 
-The motivation for SimpleX chat is [presented here](./simplex.md).
+**The world's most private and secure chat** - open-source, decentralized, and without global identities of any kind.
 
-SimpleX chat prototype is a thin terminal UI on top of [SimpleXMQ](https://github.com/simplex-chat/simplexmq) message broker that uses [SMP protocols](https://github.com/simplex-chat/simplexmq/blob/master/protocol).
+SimpleX chat prototype is a thin terminal UI on top of [SimpleXMQ](https://github.com/simplex-chat/simplexmq) message broker that uses [SMP protocols](https://github.com/simplex-chat/simplexmq/blob/master/protocol). The motivation for SimpleX chat is [presented here](./simplex.md). See [simplex.chat](https://simplex.chat) website for chat demo and the explanations of the system and how SMP protocol works.
 
-See [simplex.chat](https://simplex.chat) website for chat demo and the explanations of the system and how SMP protocol works.
-
-### Quick installation
+### :zap: Quick installation
 
 ```sh
 curl -o- https://raw.githubusercontent.com/simplex-chat/simplex-chat/master/install.sh | bash
 ```
 
-![simplex-chat](./images/connection.gif)
+Once the chat client is installed, simply run `simplex-chat` from your terminal.
 
-### Welcome word
+### :wave: Welcome
 
-**We are building the most private and secure chat in the world** - open-source, decentralized, and without global identities of any kind. If you would like to support it, you can do so in the following ways:
+**We are building the world's most private and secure chat**. If you would like to support it, you can do so in the following ways:
 
-- 🌟 **Star it on GitHub** - it's a great feedback you like it and helps us raise the visibility of the project.
+- 🌟 **Star it on GitHub** - starring the repo helps us raise the visibility of the project.
```

Comment:
my grammar sense may be off here, but I read it as "*us* starring the repo [as in activity done by us] helps ....", maybe adjust the wording here? Not sure

Reply:
Fair point - _starring the repo helps raise the visibility of the project_ sounds better